### DISCUSSION TITLE
Which baseline_table.txt used for preparing list of interferogram

### DISCUSSION THREAD

**Discussion by rofiaini** [gmtsar/gmtsar#905] [Q&A]
hai everyone, I want to ask about which baseline_table.txt used for run select_pairs.csh baseline_table.dat. I read two tutorials and got confused. 
[Tutorial1.pdf](https://github.com/gmtsar/gmtsar/files/15046024/Tutorial1.pdf)
[Tutorial2.pdf](https://github.com/gmtsar/gmtsar/files/15046026/Tutorial2.pdf)

1. While I run "preproc_batch_tops.csh data.in dem.grd 1", baseline_table.dat locate in ../F1/raw/. In tutorial 1, baseline_table.dat locate in folder F1 then copy to folder raw, so I assumed that output folder different with me. Meanwhile, in tutorial 2 doesn't which folder that baseline_table.dat, but the baseline_table.dat move to mv baseline_table.dat ../. So I assumed in tutorial 2, baseline_table.dat locate in folder ../F1/raw/ then move to ../F1/? In tutorial 2, also state that baseline_table.dat that move will use for next step.
2. based on point 1. When I run preproc_batch_tops.csh data.in dem.grd 2, new baseline_table.dat (read: baseline_table.dat_2) locate in my folder ../F1/raw/ while baseline_table.dat_1 locate in ../F1/. My ldirectory of baseline_table.dat_1 same with tutorial 2, while tutorial 1 in folder ../F1/ produce new baseline_table.dat or baseline_table.dat_2. 
3. next step, both tutorial 1 and 2 run "select_pairs.csh baseline_table.dat" in folder F1, so my question is the baseline_table.dat that used for is the first one? (baseline_table.dat_1 or output from "preproc_batch_tops.csh data.in dem.grd 1"?

**Comment by Xiaohua-Eric-Xu** [comment]
I am bit unsure what you mean here. baseline_table.dat is just a table that can be generated either by preproc_batch_tops[_esd].csh or get_baseline_table.csh. Once generated, you can put it anywhere...

**Reply by rofiaini** [reply]
baseline_table.dat for "select_pairs.csh baseline_table.dat 50 150". When run "preproc_batch_tops.csh data.in dem.grd 1" produce baseline _table.dat with master chosen from default right? While run "preproc_batch_tops.csh data.in dem.grd 1" use master that I select manually, the master will put on first line in data.in so in baseline_table.dat the master position will be in first line. Am I free to choose which baselina_table.dat to use for running "select_pairs.csh baseline_table.dat 50 150"?

**Reply by Xiaohua-Eric-Xu** [reply]
you need to use the one with mode 2 where you selected the master image, thought the differences after calculation is small.

**Reply by rofiaini** [reply]
oke, thank you
